### PR TITLE
Example: ECS FireLens integration

### LINKED ISSUE #705: Example on Data-Prepper ECS Firelens integration
**Is your feature request related to a problem? Please describe.**
Add an example on Data-Prepper ECS Firelens integration just as for logstash:

https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/logstash

**Describe the solution you'd like**

**Describe alternatives you've considered (Optional)**

**Additional context**

### PR DESCRIPTION
**Pull request by chenqi0805**
### Description
This PR adds a nginx example to demo the integration of data-prepper service with [ECS FireLens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html).
 
### Issues Resolved
#705 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed with a real name per the DCO

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by dlvenable — CHANGES_REQUESTED** [review]
Thanks for creating this example! This is a useful scenario to have documented.

File: examples/log-ingestion-ecs-firelens/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+where `<Public IP>` is exposed by the ECS task `Networking` and observe the grokked records output from data-prepper `stdout` such as:
+
+```
+{"date":1.638805848186957E9,"container_name":"app","source":"stdout","log":"70.113.36.5 - - [06/Dec/2021:15:50:48 +0000] \"GET / HTTP/1.1\" 200 615 \"-\" \"curl/7.64.1\" 
+\"-\"","container_id":"f4b6c7a71bbf49aea7b1c5c6a6bd9aa9-527074092","ecs_cluster":"firelens-dataprepper","ecs_task_arn":"arn:aws:ecs:us-east-1:531516510575:task/firelens-d
+ataprepper/f4b6c7a71bbf49aea7b1c5c6a6bd9aa9","ecs_task_definition":"firelens-example-data-prepper:1","request":"/","auth":"-","ident":"-","response":"200","bytes":"615","
+clientip":"70.113.36.5","verb":"GET","httpversion":"1.1","timestamp":"06/Dec/2021:15:50:48 +0000"}
+```
+### References:
+* To create a task definition using the classic ECS console, please refer [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-task-definition-classic.html).
```
[the remark this answers is not in the document]
Comment (chenqi0805) [reply examples/log-ingestion-ecs-firelens/README.md:35]: For some reason, ECS new console does not have `configure via JSON` option for create task definition.

[the remark this answers is not in the document]
Comment (chenqi0805) [reply examples/log-ingestion-ecs-firelens/README.md]: @dlvenable Despite that, in the updates, I still included the new console link.

**Review by dlvenable — APPROVED** [review]
Thanks! It looks good.

**Review by dinujoh — APPROVED** [review]

**Review by sshivanii — APPROVED** [review]

**Pull request merged**